code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -56,14 +56,15 @@
 
 	new /obj/item/cartridge/hos(src)
 	new /obj/item/radio/headset/heads/hos(src)
+	new /obj/item/radio/headset/heads/hos/alt(src) // SKYRAT EDIT ADDITION
 	new /obj/item/storage/bag/garment/hos(src)
 	new /obj/item/storage/lockbox/medal/sec(src)
 	new /obj/item/megaphone/sec(src)
 	new /obj/item/holosign_creator/security(src)
 	new /obj/item/storage/lockbox/loyalty(src)
 	new /obj/item/storage/box/flashbangs(src)
 	new /obj/item/shield/riot/tele(src)
-	new /obj/item/storage/belt/security/peacekeeper/full(src) //SKYRAT EDIT CHANGE - SEC_HAUL
```

Comment:
I don't really like this change. Yeah, being able to change it into the peacekeeper belt aesthetically is nice. But the Peacekeeper belt had features the normal sec belt didn't have. Most notably the holster mechanic, which was really nice and satisfying. With this you're basically removing this feature.

Reply:
The Holster mechanic was designed for a Security team that was packing pistols 24/7. They no longer do, so they no longer need it.

Reply:
wouldnt the disabler count as a pistol?

Reply:
disablers are not pistols, pistols are a specific subcategory of ballistic weapons in the code.